doc/ADMIN.md

```diff
@@ -3,6 +3,7 @@
 ## How to backup your server with this app?
```

Comment:
(we should really rework this file, it's been marked as obsolete since a while, maybe some stuff to move to DESCRIPTION.md, maybe others to PRE_INSTALL.md, maybe other to be kept in ADMIN.md)

Reply:
TBH I don’t feel good enough to edit this doc… Maybe that could be the topic of another PR.